include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1107,42 +1185,55 @@ struct bt_le_per_adv_param {
 			BT_GAP_ADV_FAST_INT_MAX_2, NULL)                                           \
 	__DEPRECATED_MACRO
 
-/** @brief GAP recommended connectable advertising
+/** @brief GAP recommended connectable advertising parameters user-initiated
+ *
```

Comment:
```suggestion
/** 
 * @brief GAP recommended connectable advertising parameters user-initiated
 *
```

The style seem to have the `@brief` on the second line